README.md

```diff
@@ -4,6 +4,9 @@ A Github Actions integration for Localize.js; use it to push the primary string
 
 ## Inputs
 
+## Input file
+This action uses the LocalizeJS cli application which requires the input file to be named `language_code.json`. 
```

Comment:
Does this mean the file must be named with a language code i.e. `en.json`, `es.json`, etc. Or that the file is literally named `language_code.json`? 

For our purposes might be easier to just assume the file is named `en.json`

Reply:
it would be en.json in our case. I will change this to be more obvious that it is a variable.